test/integration/users.test.js

```diff
@@ -186,6 +186,68 @@ describe("Users", function () {
     });
   });
 
+  describe("PATCH /add-default-status", function (done) {
+    beforeEach(async function () {
+      await addUser(userData[1]); // status does not exists user
+      await addUser(userData[0]); // status active user
```

Comment:
Create variables with meaningful names rather than comments
Comments should only be used to explain why rather than what